src/ruggeduino.cpp

```diff
@@ -47,6 +49,33 @@ void command_mode(int mode) {
   pinMode(pin, mode);
 }
 
+void command_ultrasound() {
+  int pulse = read_pin();
+  int echo = read_pin();
+
+  // config pins to correct modes
+  pinMode(pulse, OUTPUT);
+  pinMode(echo, INPUT);
+
+  // provide pulse to trigger reading
+  digitalWrite(pulse, LOW);
+  delayMicroseconds(2);
+  digitalWrite(pulse, HIGH);
+  delayMicroseconds(5);
```

Comment:
Question: Are `2` and `5` magic numbers which work, or are they specific as part of the protocol?

Reply:
This comes from the Arduino ultrasound example https://docs.arduino.cc/built-in-examples/sensors/Ping/